network-api/networkapi/wagtailpages/pagemodels/buyersguide/call_to_action.py

```diff
@@ -90,26 +68,14 @@ class Meta(TranslatableMixin.Meta):
     def __str__(self):
         return self.title
 
-    def get_target_url(self):
-        if self.link_target_url:
-            return self.link_target_url
-        else:
-            return self.link_target_page.url
-
-    def clean(self):
-        errors = {}
-        duplicate_link_target_error = ErrorList(["Please select a page OR enter a URL for the link (choose one)"])
-        # If user enters both link URL and link page:
-        if self.link_label and (self.link_target_url and self.link_target_page):
-            errors["link_target_url"] = duplicate_link_target_error
-            errors["link_target_page"] = duplicate_link_target_error
-        # If user enters link label but no page or URL to link to:
-        elif self.link_label and (not self.link_target_page and not self.link_target_url):
-            errors["link_target_url"] = duplicate_link_target_error
-            errors["link_target_page"] = duplicate_link_target_error
-        # If user does not enter a label but has set link URL or page.
-        elif not self.link_label and (self.link_target_page or self.link_target_url):
-            errors["link_label"] = ErrorList(["Please enter a label for the link"])
+    def link_label(self):
+        if self.link:
+            link_block = self.link[0]
+            return link_block.value["label"]
+        return ""
```

Comment:
For use in `search_fields` and the list view of the CMS

Reply:
Thanks @danielfmiranda, I was just looking through to find where we were using `link_url` and `link_label`. Nice!